.github/workflows/crowdin-pull.yml

```diff
@@ -63,6 +63,7 @@ jobs:
           push_translations: false
           # Pull request options
           create_pull_request: false
+          command_args: '--verbose'
```

Comment:
You can remove these `--verbose` flags. I'm assuming they were just for testing.

Reply:
Removed